examples/simple_ecat.c

```diff
@@ -0,0 +1,76 @@
+#include <ingenialink/ingenialink.h>
+#include <ingenialink/registers.h>
+#include <time.h>
+#include <stdio.h>
+
+
+int main(int argc, const char *argv[])
+{
+	il_net_t *net = NULL;
+	il_servo_t *servo = NULL;
+	char* ifname = "\\Device\\NPF_{XXX}";
+	char* dict_path = "XXX";
+	int port = 1061;
+	int slave = 1;
+	int use_eoe = 1;
+	int r;
+	r = il_servo_connect_ecat(IL_NET_PROT_ECAT, ifname, &net, &servo, dict_path, port, slave, use_eoe);
+
+	if (r < 0) {
+		fprintf(stderr, "CONNECTION FAIL %d\n", r);
+		return -1;
+	}
+
+	int32_t actual_position_buf;
```

Comment:
Follow [style guide ](https://github.com/ingeniamc/ingenia-style-guide/blob/master/c_cpp_style.md#naming) for naming.